shared/components/StudyInfo/LearningPeriod.tsx

```diff
@@ -46,41 +47,28 @@ export class LearningPeriod extends React.Component<LearningPeriodProps> {
     const {
       alku,
       loppu,
-      nimi,
       tyyppi,
       tyopaikallaJarjestettavaKoulutus,
       selite,
       workplaceSelite,
       ajanjaksonTarkenne,
-      jarjestajanEdustaja
+      jarjestajanEdustaja,
+      muutOppimisymparistot
     } = learningPeriod
 
     const vastuullinenTyopaikkaOhjaaja =
-      tyopaikallaJarjestettavaKoulutus?.vastuullinenTyopaikkaOhjaaja
+      tyopaikallaJarjestettavaKoulutus.vastuullinenTyopaikkaOhjaaja
```

Comment:
const {vastuullinenTyopaikkaOhjaaja, keskeisetTyotehtavat} = tyopaikallaJarjestettavaKoulutus

Reply:
yes, that's cleaner